icechunk-python/tests/test_can_read_old.py

```diff
@@ -192,7 +192,7 @@ async def test_icechunk_can_read_old_repo():
     ]
     assert sorted(
         [p async for p in store.list_dir("group2/group3/group4/group5/inner")]
-    ) == ["zarr.json"]
+    ) == ["c", "zarr.json"]
```

Comment:
Does anybody understand why I need to change this? This change makes sense, I think, but why it was passing before and not now with this apparently unrelated change?